ALUM/ALUM/Views/SignUpPage/SignUpSetUp.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpSetUp.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/27/23.
+//
+
+import SwiftUI
+
+struct SignUpSetUp: View {
+    @ObservedObject var viewModel: SignUpPageViewModel
+    
+    var body: some View {
+        ZStack {
+            Color("ALUM White 2").edgesIgnoringSafeArea(.all)
+                .overlay(alignment: .top) {
+                    ProgressBarComponent(nodes: 3, filledNodes: 0, activeNode: 1)
+                        .frame(alignment: .top)
+                        .frame(maxWidth: .infinity)
+                        .padding()
+                        .background(Color.white.ignoresSafeArea(edges: .top))
+                        .frame(maxWidth: .infinity)
```

Comment:
action - same comment from SignUpJoinAs to change the structure of the Z, V, H stacks.